cryptol-remote-api/server-not-threaded.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/cryptol-remote-api/cryptol-remote-api.cabal b/cryptol-remote-api/cryptol-remote-api.cabal
```

Comment:
Do you need this `.patch` file if the use of `-threaded` is controlled by a `cabal` flag?

Reply:
whoops - thanks! =)